linux_os/guide/system/permissions/files/dir_perms_world_writable_root_owned/ansible/shared.yml

```diff
@@ -4,64 +4,11 @@
 # complexity = low
 # disruption = medium
 
-- name: "{{{ rule_title }}} - Define Excluded (Non-Local) File Systems and Paths"
-  ansible.builtin.set_fact:
-    excluded_fstypes:
-      - afs
-      - ceph
-      - cifs
-      - smb3
-      - smbfs
-      - sshfs
-      - ncpfs
-      - ncp
-      - nfs
-      - nfs4
-      - gfs
-      - gfs2
-      - glusterfs
-      - gpfs
-      - pvfs2
-      - ocfs2
-      - lustre
-      - davfs
-      - fuse.sshfs
-    excluded_paths:
-      - dev
-      - proc
-      - run
-      - sys
-    search_paths: []
-    world_writable_dirs: []
-
-- name: "{{{ rule_title }}} - Find Relevant Root Directories Ignoring Pre-Defined Excluded Paths"
-  ansible.builtin.find:
-    paths: /
-    file_type: directory
-    excludes: "{{ excluded_paths }}"
-    hidden: true
-    recurse: false
-  register: result_relevant_root_dirs
-
-- name: "{{{ rule_title }}} - Include Relevant Root Directories in a List of Paths to be Searched"
-  ansible.builtin.set_fact:
-    search_paths: "{{ search_paths | union([item.path]) }}"
-  loop: "{{ result_relevant_root_dirs.files }}"
-
-- name: "{{{ rule_title }}} - Increment Search Paths List with Local Partitions Mount Points"
-  ansible.builtin.set_fact:
-    search_paths: "{{ search_paths | union([item.mount]) }}"
-  loop: '{{ ansible_mounts }}'
-  when:
-    - item.fstype not in excluded_fstypes
-    - item.mount != '/'
+{{{ ansible_create_list_of_local_paths(list_name="search_paths") }}}
```

Comment:
I think that the fail of the test scenario `world_writable_dir_on_nonlocal_fs.fail.sh` on the CI is caused by the invocation of `systemctl` command in this scenario, which hits the problem that there is no Dbus in the container back end environment that we use in CI.

```
+ systemctl restart nfs-server
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
```

When executed local with a virtual machine back end the test is passing.

```
[jcerny@fedora scap-security-guide{pr/10912}]$ python3 tests/automatus.py rule --remediate-using ansible --libvirt qemu:///system ssgts_rhel9 dir_perms_world_writable_root_owned
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/work/git/scap-security-guide/logs/rule-custom-2023-07-27-1438/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_dir_perms_world_writable_root_owned
INFO - Script world_writable_dir_on_nonlocal_fs.fail.sh using profile (all) OK
INFO - Script world_writable_dir_owned_by_uid_2.fail.sh using profile (all) OK
INFO - Script all_dirs_ok.pass.sh using profile (all) OK
```

I think the problem isn't related to the contents of this PR.